src/main/java/team/gif/robot/Globals.java

```diff
@@ -3,4 +3,5 @@
 // Should be used sparingly
 public abstract class Globals {
     public static boolean indexerEnabled = true;
+    public static boolean inputValue;

```

Comment:
Is this necessary? Looks like this is declared twice, once here and again in robot.java

Reply:
No, we don't need it. I forget to delete that line.